source/post-toot/src/post-toot.py

```diff
@@ -1 +1,42 @@
-code="in_dev"
+from mastodon import Mastodon
+import boto3
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+toots_table_name = os.getenv('TOOTS_TABLE_NAME')
+s3 = boto3.resource('s3')
+dynamodb = boto3.resource('dynamodb')
+table = dynamodb.Table(toots_table_name)
+mastodon = Mastodon(
+    access_token = os.environ.get('MASTODON_ACCESS_TOKEN'),
+    api_base_url = 'https://techhub.social'
+)
+
+def post_toot(record):
+    s3_data = record["s3"]
+    bucket_name = s3_data["bucket"]["name"]
+    file_name = s3_data["object"]["key"]
+    id = int(file_name[0:-4])
+    response = table.get_item(
+    Key={
+        'id' : id 
+        }
+    )
+    username = response['Item']['username']
+    in_reply_to_id = response['Item']['id']
+    status = username + " here is your AI-ART!"
+    s3.meta.client.download_file(bucket_name , file_name, '/tmp/'+file_name)
+    media_file = '/tmp/'+file_name
+    
```

Comment:
can you split this into functions?
This is a lot of stuff.

maybe this are separate functions

```
 s3_data = record["s3"]
 bucket_name = s3_data["bucket"]["name"]
 file_name = s3_data["object"]["key"]
   ```

   ```
id = int(file_name[0:-4])
    response = table.get_item(
    Key={
        'id' : id 
        }
    )
    username = response['Item']['username']
   ```

(is in_reply_to_id not the same as id)

than download of the file

and sending